homeassistant/components/bond/light.py

```diff
@@ -231,3 +279,32 @@ async def async_turn_off(self, **kwargs: Any) -> None:
         _LOGGER.debug("Fireplace async_turn_off called with: %s", kwargs)
 
         await self._hub.bond.action(self._device.device_id, Action.turn_off())
+
+    async def async_set_state_belief(self, **kwargs: Any) -> None:
+        """Set the belief state of the fireplace."""
+        brightness = kwargs.get(ATTR_BRIGHTNESS)
+        if brightness is not None:
+            if not self._device.supports_set_brightness():
+                raise TemplateError(
+                    Exception("This device does not support brightness")
+                )
+            if brightness == 0:
+                await self._hub.bond.action(
+                    self._device.device_id, Action.set_power_state_belief(False)
+                )
+            else:
+                await self._hub.bond.action(
+                    self._device.device_id,
+                    Action.set_brightness_belief(round((brightness * 100) / 255)),
+                )
+        else:
+            state = kwargs.get(ATTR_POWER_STATE)
+            if state is None:
+                raise TemplateError(
+                    Exception(
+                        f"Either {ATTR_BRIGHTNESS} or {ATTR_POWER_STATE} is required.  Neither were provided."
+                    )
+                )
+            await self._hub.bond.action(
+                self._device.device_id, Action.set_power_state_belief(state)
+            )
```

Comment:
```suggestion
```

You should only need it in the parent entity since it will be inherited by the children classes

Reply:
The BondFireplace doesn't inherit it.  and the command is slightly different for them.